fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/models/FedoraResource.java

```diff
@@ -156,18 +150,18 @@ RdfStream getTriples(final IdentifierConverter<Resource, FedoraResource> idTrans
                          final Set<? extends TripleCategory> contexts);
 
     /**
-     * Get the JCR Base version for the node
+     * Get the base version for the node
      *
      * @return base version
      */
-    public Version getBaseVersion();
+    FedoraResource getBaseVersion();
```

Comment:
What I'd like to see is these methods renamed to reflect the memento spec. For instance, we might have `getTimegates` instead of `getVersions`, but that's for a separate effort (ideally after the versioning spec has been finalized.